doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -66,9 +66,27 @@ For an application that uses P-GPS, the following options must be configured for
 * :kconfig:option:`CONFIG_FCB`
 * :kconfig:option:`CONFIG_SETTINGS_FCB`
 * :kconfig:option:`CONFIG_DATE_TIME`
-* :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT`
-* :kconfig:option:`CONFIG_IMG_MANAGER`
-* :kconfig:option:`CONFIG_MCUBOOT_IMG_MANAGER`
+
+To store predictions, the P-GPS library requires space in the flash memory.
+Enable the following option to use a dedicated partition:
+
+* :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_PARTITION`
+
+If the flash memory storage for your application is insufficient to use a dedicated partition, but it does use MCUboot for FOTA updates, you can use the following option:
+
+* :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_MCUBOOT_SECONDARY`
+
+However, if you also enable FOTA updates of MCUboot itself, enabling both :kconfig:option:`CONFIG_SECURE_BOOT` and :kconfig:option:`CONFIG_BUILD_S1_VARIANT`, do not use the previous option.
+The P-GPS library will prevent the MCUboot update from fully completing, so the primary bootloader will revert MCUboot to its previous image.
+
+A third option is to define:
```

Comment:
Hi! In which scenario do you have to use this option?  (CONFIG_NRF_CLOUD_PGPS_STORAGE_CUSTOM)
This is not clear from the text.

Reply:
I'll add  more context.

`If you do not want or need to use MCUboot, and want complete control over where in flash to store P-GPS data, a third option is to define:`